reactioncommerce/reaction
# (fix): also send notification to shop owners upon order creation

Resolves #4211   
Impact: minor
Type: bugfix

## Issue
When an order is created, it's always the case that only the marketplace admin is notified even when the order contains items sold by other merchants. There is currently no functionality to support such case.

## Solution
Update code for notifications such that we loop through the items of the order, check who owns the shop of each item, and send each owner a notification.

## Breaking changes
None

## Testing
1. Activate the marketplace feature.
2. Invite a new shop owner/merchant.
3. Activate the new shop created.
4. Click 'Get Started' from the e-mail invite. While logged in as the merchant, update the address of the shop and setup Stripe payment. Create and publish a product. 
5. As another user or possibly guest, create an order for that merchant product along with product/s from the primary shop.
6. Possibly, more merchants can be created and order can have items from different merchants.
7. Admin and all merchants involved in the order should receive notification.
